### ISSUE TITLE
bing.com/maps

### ISSUE THREAD

**Issue by GuardianMajor** [AdguardTeam/AdguardFilters#12898]
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behavior**: 

I am currently using `Adguard Spyware Filters` on uBlock and when trying to get directions on Bing Maps, it will fail to do anything (goes into an endless waiting loop) because of this filter `||bing.com/fd/ls/` and setting an exception or simply removing your list will fix the issue.

* **Expected behavior**: 

It should be able to provide directions and display the content normally.

***Steps to reproduce the problem***:

* Goto www.bing.com/maps
* Enter an address of your destination
* Then select directions
* Choose to here
* Enter your starting point
* It begins to process (dancing blue dots)
* (optionally) Either click go or set any other options like "leave at"
* See dancing blue dots indicating it is trying to process but nothing happens
* See the logger and see that it is continuously blocking `xhr` with this filter `||bing.com/fd/ls/`

***System configuration***

**Filters:**

Tested with all filters disabled and using only `Adguard Spyware Filters` and no change. Removing the list, fixes the issue as this is the only filter list on this system that has this filter `||bing.com/fd/ls/`

[//]: # (Please enter the correct values for your case to the table below)

Information                            | Value
---                                    | ---
Operating system:                      | Windows 10 Pro x64 (not that it matters)
Operating system version (Android/iOS) | 1709 (again not that it matters)
Browser:                               | 56.0.2 (64-bit)
AdGuard version:                       | N/A (filter only)
Filters enabled:                       | Adguard Spyware Filters
AdGuard mode (Android only):           | N/A
Filtering quality (Android only):      | N/A
HTTPS filtering (Android only):        | N/A
Simplified filters (iOS only)          | N/A
AdGuard DNS:                           | N/A
Stealth mode options (Windows only)    | N/A
Helpdesk ID (if exists):               | N/A

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

**Comment by Alex-302** [comment]
Not reproduced. 
@AdamWr @BlazDT please take a look.

**Comment by Alex-302** [comment]
Please check
`@@||bing.com/fd/ls/GLinkPing.aspx?`

**Comment by AdamWr** [comment]
It works fine on my end.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/37421225-11a5e44e-27b9-11e8-90a6-533f38b9758d.png)

</details>

**Comment by GuardianMajor** [comment]
@Alex-302 As you can see here, it is not working and @AdamWr did you use the above proposed exception or without it?

![image](https://user-images.githubusercontent.com/5274181/37427836-bc825776-2787-11e8-9d19-e45689cdbcdb.png)

**Comment by GuardianMajor** [comment]
@Alex-302 Just a follow-up, that exemption does nothing to fix the issue either. Thank you.

**Comment by BlazDT** [comment]
Not reproduced.

Please try this rule

`@@||bing.com/fd/ls/lsp.aspx`

**Comment by GuardianMajor** [comment]
@BlazDT I am aware of how to make an exception to the blocking rules, that's not the point. The point is why is this causing it to break, not how to write an exemption code. Since you claim that you can't reproduce this and works fine for you. It can't be that it works fine on one machine while blocking but needs to be exempted on another to work.

**Comment by GuardianMajor** [comment]
So in the spirit that you couldn't reproduce it, I started looking to see if something else is causing it. While it doesn't show up in the logger, as you have seen earlier, I noticed on the NETWORK panel of the debug tools that resources belonging to virtualearth were being marked as failed to load.

Added this: `@@||virtualearth.net/webservices/*$script,domain=bing.com,important` and it went back to working fine, even though still there is no reference to virtualearth at all in the uBlock logger, which is odd as there is nothing actually blocking this anywhere, so makes absolutely no sense but none the less this line fixed it. I need to figure out why.
